camayoc/tests/qpc/cli/test_sources.py

```diff
@@ -756,6 +762,42 @@ def test_add_with_cred_hosts_exclude_file(
     )
 
 
+@pytest.mark.parametrize('source_type, hosts, exclude_hosts',
+                         INVALID_SOURCE_TYPE_HOSTS_WITH_EXCLUDE_HOSTS)
+def test_add_exclude_hosts_negative(isolated_filesystem, qpc_server_config,
+                                    source_type, hosts, exclude_hosts):
+    """Add a source with cred and hosts and exclude a host.
+
+    :id: d0f24930-f05d-46c3-af36-493228e01822
+    :description: Add a source entry providing the ``--name``, ``--cred``,
+        ``--hosts``, and ``--exclude-hosts`` options.
+    :steps: Run ``qpc source add --name <name> --cred <cred>
+        --hosts <hosts> --exclude-hosts <excludedhosts> --type <type>``
+    :expectedresults: A new source entry is created with the data provided as
+        input.
```

Comment:
need to update the doc string to reflect what the test is doing

Reply:
you can do multi line steps like this:
https://github.com/quipucords/camayoc/blob/37d70c8cc2608e3937d749802eb1f8c9c622cb4a/camayoc/tests/qpc/api/v1/sources/test_network_sources.py#L42